data/apps/working-copy.json

```diff
@@ -0,0 +1,69 @@
+{
+	"info": {
+		"identifier": "working-copy",
```

Comment:
I know I did not specify that anywhere but it would be great if you could use 2 space indentation.

Reply:
Sure thing. I will clean this up when adding additional commands.

Reply:
Would you prefer I merge this PR and you create a separate one for the rest or do you want to do it in this one?

Reply:
I just fixed the indentation. It would be nice if you merged it as I think it will take me a little while to get through the remaining 15 commands.